lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -120,7 +120,8 @@ def test_query(
         }
 
         index = self.build_index(
-            server_config, self.date_range(self.payload["start"], self.payload["end"])
+            server_config,
+            dates=self.date_range(self.payload["start"], self.payload["end"]),
         )
```

Comment:
See previous comment.

Ditto for the other calls to `build_index` in this module.